zipline/finance/risk/cumulative.py

```diff
@@ -291,9 +290,9 @@ def update(self, dt, algorithm_returns, benchmark_returns, account):
             raise Exception(message)
 
         self.update_current_max()
-        self.metrics.benchmark_volatility[dt] = \
+        self.metrics.benchmark_volatility.values[dt_loc] = \
```

Comment:
You're right, as well as prevent the case of accidentally creating copy as we had discussed.